src/CookieAttributes.ts

```diff
@@ -1,36 +1,51 @@
-export interface CookieAttributes {
+export type CookieAttributes = BaseCookieAttributes & SameSiteCookieAttributes
+
+export interface BaseCookieAttributes {
     /**
      * A number will be interpreted as days from time of creation
      */
-    expires?: Date | number;
+    expires?: Date | number
```

Comment:
Why remove the semicolons? I would prefer keeping them.

Reply:
My bad. I didn't notice there is no `prettierrc` in the project.